README.md

```diff
@@ -94,11 +94,30 @@ $ sudo journalctl -t cc-runtime
 
 ### Enabling debug for various components
```

Comment:
What do you think to changing this title to:
  `Enabling and viewing debug for other components`
to cover the fact we now show enabling and viewing, and we cover the runtime separately in the above section ?

Reply:
I take your point but this section also enables debug for the runtime. I'll rework both sections to make this clearer...